### PR TITLE
[Feature] Adds Documentation For Union Types

### LINKED ISSUE #1870: [Feature Request] GraphQL Documentation for Union
**Is your feature request related to a problem? Please describe.**
The GraphQL documentation portion of Insomnia does not resolve `Union` data types. Drilling into one of these types always produces a `no description` value to be displayed.

**Describe the solution you'd like**
It would be great if when selecting a `Union` data type, any types that make up that union are presented in the same way that `Fields` is presented.

**Describe alternatives you've considered**
Right now there is no alternative solution except to look at the raw schema or use a different GraphQL client that parses this properly.

### PR DESCRIPTION
**Pull request by elemanhillary-zz**
Closes [#1870](https://github.com/Kong/insomnia/issues/1870)
Screen Shot 2020-01-10 at 4 58 30 PM

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/graph-ql-explorer/graph-ql-explorer-type.js [hunk 1]
```diff
@@ -29,6 +29,32 @@ class GraphQLExplorerType extends React.PureComponent<Props> {
...
+    if (
+      Object.prototype.toString.call(type) !== '[object GraphQLUnionType]' &&
+      Object.prototype.toString.call(type.getTypes) !== '[object Function]'
+    ) {
+      return null;
+    }
+    const types = type.getTypes();
+    return (
+      <React.Fragment>
+        <h2 className="graphql-explorer__subheading">Types</h2>
```
Comment (gschier) [inline packages/insomnia-app/app/ui/components/graph-ql-explorer/graph-ql-explorer-type.js]: Can we update this heading to be "Possible Types" to match what GraphiQL shows?

**Review by gschier — COMMENTED** [review]
Thanks for taking this one @elemanhillary! just one comment

**Comment by gschier** [comment]
Oh, also. It looks like the CI is failing on here due to a Flow issue. Let me know if you need help resolving that.

**Comment by elemanhillary-zz** [comment]
> Oh, also. It looks like the CI is failing on here due to a Flow issue. Let me know if you need help resolving that.

let me sort it out, if i fail i will let you know

**Comment by barthez** [comment]
@elemanhillary Thanks for the PR, I would love to have that feature. Do you need any help solving CI failures?

**Pull request merged**